File: backend/src/main/java/dto/cmd/server/LeftGameCmd.java
package dto.cmd.server;

import dto.DTOCode;
import dto.DTOs;
import io.netty.buffer.ByteBuf;


@DTOCode(DTOs.LEFT_GAME)
public class LeftGameCmd extends ServerCommand {

    @Override
    protected void writeMembers (ByteBuf byteBuf) {

    }


    @Override
    protected void readMembers (ByteBuf buffer) {

    }
}

File: frontend/Assets/Scripts/DataTransfer/Server/LeftGameCmd.cs
using System;
using System.IO;
using Attributes;


namespace DataTransfer.Server {

    [DTO (DTOCode.LeftGame)]
    public class LeftGameCmd : ServerCommand {

        public override void ReadMembers (BinaryReader reader) {
            throw new NotImplementedException ();
        }


        public override void Execute () {
            throw new NotImplementedException ();
        }

    }

}